rust/stackable-cockpit/src/platform/release/spec.rs

```diff
@@ -23,6 +28,17 @@ pub enum Error {
     #[snafu(display("failed to parse operator spec"))]
     OperatorSpecParse { source: operator::SpecParseError },
 
+    /// This error indicates that parsing a string into a path or URL failed.
+    #[snafu(display("failed to parse '{path_or_url}' as path/url"))]
```

Comment:
We have a convention of using the `Debug` impl of `String` so that is quotes, but also escapes inner quotes.


```suggestion
    #[snafu(display("failed to parse {path_or_url:?} as path/url"))]
```

Reply:
https://github.com/stackabletech/stackable-cockpit/pull/379/commits/5e66494f1cec9d166883361f938a52ee3792e627